kolibri/core/logger/csv_export.py

```diff
@@ -180,6 +183,19 @@ def exported_logs_info(request):
 
 
 def download_csv_file(request, log_type):
+    locale = get_language_from_request(request)
+    translation.activate(locale)
+
+    csv_translated_filenames = {
+        "session": pgettext(
+            "Default name for the exported CSV file with content session logs. Please keep the underscores between words in the translation",
+            "content_session_logs.csv",
+        ),
+        "summary": pgettext(
+            "Default name for the exported CSV file with content summary logs. Please keep the underscores between words in the translation",
+            "content_summary_logs.csv",
+        ),
+    }
```

Comment:
Rather than expecting translators to write valid file names, recommend using Django's [`slugify`](https://docs.djangoproject.com/en/3.0/ref/utils/#django.utils.text.slugify) function and doing something like:

```python
slugify(
  pgettext(
    "Name for the exported CSV file with content session logs",
    "content session logs"
  )
) + '.csv'
```

Reply:
Done